server/@types/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace Express {
+  interface Request {
+    userId: string;
```

Comment:
Is this necessary? You should be able to `import { Request } from "express"` and use userid as a param